plugins/scaffolder-backend/src/scaffolder/actions/builtin/catalog/getEntityByRef.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CatalogApi } from '@backstage/catalog-client';
+import { createTemplateAction } from '../../createTemplateAction';
+import yaml from 'yaml';
+
+const id = 'catalog:getEntityByRef';
```

Comment:
Wonder if we could make this something simpler, like `catalog:fetch`, as long as it's still intuitive. Otherwise we typically `kebab-case` the action IDs

Reply:
Good idea, will change it when I find the time!